packages/fast-components-react-msft/src/flyout/flyout.schema.ts

```diff
@@ -0,0 +1,74 @@
+export default {
+    $schema: "http://json-schema.org/schema#",
+    title: "Flyout",
+    description: "A flyout component's schema definition.",
+    type: "object",
+    id: "@microsoft/fast-components-react-msft/flyout",
+    properties: {
```

Comment:
does this need a `formPluginId`?

Reply:
I've been adding them to schemas for the tooling, so I would say both a `formPluginId` and a `pluginId` would be great